### PR TITLE
Module 1: Mobile first

### PR DESCRIPTION
**Pull request by henry-dura**
I have designed a mobile version of my portfolio page using a max-width of 375px for mobile.

### CODE REVIEW

**Review by anagudelogu — COMMENTED** [review]
Hi @henry-dura, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that doesn't use the correct README [template](https://github.com/microverseinc/readme-template). Please update it and submit your project again.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by amejid — COMMENTED** [review]
Hi @henry-dura, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

- [ ] You have submitted a project, that does not follow the README [template](https://github.com/microverseinc/readme-template) provided by Microverse. Kindly update it accordingly, the only optional section from the README is `FAQ` section.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>portfolio mobile first</title>
+  </head>
+  <body>
+    <div class="container">
+      <nav>
+        <p id="name">Henry Ik</p>
+        <img src="images/Union.png" alt="menu" />
```
Comment (tufoinnkuo10) [inline index.html]: - [x] for semantic purposes, kindly put only the hamburger icon inside the nav tag. nice work

File: index.html [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>portfolio mobile first</title>
+  </head>
+  <body>
+    <div class="container">
```
Comment (tufoinnkuo10) [inline index.html:11]: - [x] please ensure that the toolbar section is inside a `header `tag not a div.[Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header) for more info

File: index.html [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+      <nav>
+        <p id="name">Henry Ik</p>
+        <img src="images/Union.png" alt="menu" />
+      </nav>
+      <main>
+        <section class="name">Hey There. I'm Henry</section>
```
Comment (tufoinnkuo10) [inline index.html]: - [x] please ensure that The container of the whole headline  section is a `section` tag . It represents a generic standalone section of the document, so use it. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) for more info.

Comment (tufoinnkuo10) [inline index.html]: - [x] please use an h1 tag for the main title. `Hey There. I'm Henry`. this help google and other search engines know about a  particular topic

File: index.html [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+        <section class="proffession">I am a Software Developer</section>
+        <section class="skill">
```
Comment (tufoinnkuo10) [inline index.html]: - [x] kindly use a `p tag` here for your into text

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @henry-dura 

There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight
✅ You used the correct Github flow
✅ Descriptive pull request message
✅ linter error checks validated
✅ Design well implemented


Required Changes ♻️
Check the comments under the review

tufoin

Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Review by bienvenuushindi — APPROVED** [review]
# STATUS: APPROVED 🟢 

## Hi @henry-dura   👋, 

Well done implementing the requested changes 👏
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



### Highlights 🎯 
:+1: No linter errors✔️
:+1: Good UI design✔️
:+1: Good PR title

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**